cmd/compute-domain-controller/computedomain.go

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2025 NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+
+	nvapi "github.com/NVIDIA/k8s-dra-driver-gpu/api/nvidia.com/resource/v1beta1"
+	nvinformers "github.com/NVIDIA/k8s-dra-driver-gpu/pkg/nvidia.com/informers/externalversions"
+)
+
+type GetComputeDomainFunc func(uid string) (*nvapi.ComputeDomain, error)
+
+const (
+	informerResyncPeriod = 10 * time.Minute
```

Comment:
choice of time constant: did any interesting considerations come to mind when you picked this? You probably made some kind of coarse-grained "not too big, not too small" estimation, and I wonder if there are any war stories associated with such choice :).

Reply:
It is the one that patrick used here and I just borrowed it:
https://github.com/kubernetes/kubernetes/blob/481cc1a39217e1c058f6b2fb7517a8d03ee40382/staging/src/k8s.io/dynamic-resource-allocation/resourceslice/resourceslicecontroller.go#L49

In a non-buggy component, it shouldn't even be needed.

Reply:
ok. thanks for the pointer!